library.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "esp32-camera",
+    "keywords": "esp32, camera",
+    "description": "esp32-camera",
+    "authors": [

```

Comment:
Please remove this "authors" block, PlatformIO will calculate it automatically. Espressif != Expressif :)